packages/fx-core/resource/package.nls.json

```diff
@@ -293,6 +293,8 @@
   "core.createProjectQuestion.apiSpecInputUrl.label": "Enter OpenAPI Description Document Location",
   "core.createProjectQuestion.OpenAIPluginDomain": "OpenAI Plugin Manifest",
   "core.createProjectQuestion.OpenAIPluginDomain.placeholder": "Enter your website domain or manifest URL",
+  "core.createaProjectQuestion.ApiKey": "Client secret for Api key in OpenAPI spec",
+  "core.createProjectQuestion.invalidApiKey.message": "Client secret is invalid. Length of secret should >= 10 and <= 128",
```

Comment:
Length -> The length
should -> should be

Reply:
Updated, thanks!